pkg/client/devops/jenkins/jenkins.go

```diff
@@ -1,16 +1,18 @@
-// Copyright 2015 Vadim Kravcenko
```

Comment:
This change is not right. I guess this file was copied from somewhere. We need to keep the original author.